unit_test_and_lint.sh

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-pylint MaxText/
+python3 -m pylint $(git ls-files '*.py')
```

Comment:
Why was this change necessary?

Reply:
It makes sure that pylint runs on all files in maxtext repo if you're using unit_test_and_lint.sh before committing code changes. Also this is how pylint.yml uses pylint, don't really see a reason why it should be different in 2 places in the same repo

Reply:
Seems fine to me! Just curious.